### ISSUE TITLE
Org sign up

### ISSUE THREAD

**Issue by Br3nda** [ServiceInnovationLab/feijoa#167]
If a user signs up, with an email that matches a record (or several records) in organisation.email, make them a member of that organisation.

**Comment by mermop** [comment]
maybe we could send an invitation?

**Comment by Br3nda** [comment]
Yes, they should verify their emails -- all the users. For now i want to avoid accidentally emailing schools tho.

**Comment by Br3nda** [comment]
once they are registered as the first user for an org, they can use #136 to invite more
